src/common/types.h

```diff
@@ -405,7 +405,6 @@ typedef struct
             file_ownership_t before_owner;
         } modify;
         struct {
-            file_info_t source;
```

Comment:
I guess we don't need this because it's the same info, since the name isn't part of our file_info_t here and we just renamed the file.